extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -162,13 +163,17 @@ const TEMPLATE = {
               tag: 'div',
               attrs: dict({
                 'role': 'button',
+                'aria-label':
+                  LocalizedStringId.AMP_STORY_AUDIO_UNMUTE_BUTTON_LABEL,
                 'class': UNMUTE_CLASS + ' i-amphtml-story-button',
               }),
             },
             {
               tag: 'div',
               attrs: dict({
                 'role': 'button',
+                'aria-label':
```

Comment:
I don't think that this will actually pass the localized string; the `LocalizedStringId` here is actually just the key for the string.  You'll need to pass it to [LocalizationService#getLocalizedString](https://github.com/ampproject/amphtml/blob/7a5e2f748c4dfd5e2e25e7496ed7d4965498791b/src/service/localization.js#L140-L158) to get the actual string.

Alternatively, you can update [simple-templates](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/simple-template.js#L73) to automatically run `aria-label` (or any other attribute) through the localization service, as it already does for the `localizedStringId` field.

Reply:
Ah got it, that makes sense - I think I'll pass the key through LocalizationService#getLocalizedString.

Reply:
Took some messing with but I decided to go with a different strategy for encapsulations sake - adding an option for `aria-labels` directly into the templating. Let me know if you think it would be better to just use a `LocalizationService` instance inside the `amp-story-system-layer.js` file